test/suites/device-virtual/main_test.go

```diff
@@ -38,19 +45,6 @@ func TestMain(m *testing.M) {
 	// make sure all services are online before starting the tests
 	utils.WaitPlatformOnline(nil)
 
-	// for local build, the interface isn't auto-connected.
-	// connect manually regardless
-	utils.SnapConnect(nil,
-		"edgexfoundry:edgex-secretstore-token",
-		deviceVirtualSnap+":edgex-secretstore-token",
-	)
-
-	// Start the service so that the default config gets uploaded to consul.
-	// Otherwise, settings that get passed using environment variables on first start get uploaded
-	// and become the default.
-	utils.SnapStart(nil, deviceVirtualService)
-	utils.WaitServiceOnline(nil, 60, deviceVirtualServicePort)
-
```

Comment:
Why are these removed?

Reply:
They have been handled by the beginning of config test now:
https://github.com/canonical/edgex-snap-testing/blob/dbc11de74b039af37f0d5731855e62659739c759/test/utils/config.go#L29-L33

Reply:
great. thanks